darts/dataprocessing/transformers/scaler.py

```diff
@@ -26,73 +31,60 @@ def __init__(self, scaler=MinMaxScaler(feature_range=(0, 1)), name="Scaler"):
             will scale all the values of a time series between 0 and 1.
```

Comment:
Perhaps you could add a sentence here saying that if this transformer is applied on multiple series, then this scaler will be deep-copied for each series.